x-element.js

```diff
@@ -21,6 +22,15 @@ export default class XElement extends HTMLElement {
     return XElement.defaultTemplateEngine;
   }
 
+  /**
+   * Declare an array of CSSStyleSheet objects to adopt on the shadow root.
+   * Note that a CSSStyleSheet object is the type returned when importing a
```

Comment:
nit: an example syntax might be more useful here

Reply:
I’ll add some additional commentary into the changes to use JSDoc 👌